grammar/asdl_cpp.py

```diff
@@ -1675,7 +1675,7 @@ def add_masks(fields, node):
 
 FOOT = r"""} // namespace LFortran::%(MOD)s
```

Comment:
Removing  `LFortran` from the `namespace` is messing with the things in `libasr`.

Reply:
With the current PR, do you see any failures in libasr?

Reply:
Nah, it doesn't fail. 
I said **if** we remove `LFortran` from `namespace` then it fails: in turn, we have to alter the content in the `Libasr`